src/DotNetWorker.Core/Context/FunctionContext.cs

```diff
@@ -34,6 +34,12 @@ public abstract class FunctionContext
         /// </summary>
         public abstract BindingContext BindingContext { get; }
 
+        /// <summary>
+        /// Gets the retry context containing information about retry acvitity for the event that triggered
+        /// the current function invocation.
+        /// </summary>
+        public abstract RetryContext RetryContext { get; }
```

Comment:
Maybe nullable?

Reply:
See above. I think we should make sure the defaults are applied and avoid a world where people have to perform `null` checks when working against the context all over the place (or worse yet, failing to do so and running into `null` ref exceptions)